docs/catalog/cloud/evaluate-using-a-service-mesh.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
+- network
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Evaluate using a service mesh
+
+## Description
+A service mesh deploys additional containers for communication, typically in a sidecar pattern, to provide more operational capabilities leading to an increase in CPU usage and network traffic. 
+
+Nevertheless, it allows you to decouple your application from these capabilities as it moves them out from the application layer, and down to the infrastructure layer.
+
+## Solution
+Carefully consider the increase in CPU usage and network traffic generated by service mesh communication components before making the decision to use one.
```

Comment:
What are the considerations a cloud engineer should be accounting for? How should they be weighing the service-mesh decision against functional requirements and emissions?